### DISCUSSION TITLE
Can MacroDroid be granted access to the android.permission.READ_LOGS permission using webadb?

### DISCUSSION THREAD

**Discussion by ghost** [yume-chan/ya-webadb#578] [Q&A]
If so, what is the correct command to grant MacroDroid access to the android.permission.READ_LOGS permission, since the command 'adb shell pm grant com.arlosoft.macrodroid android.permission.READ_LOGS' does not work?

**Comment by yume-chan** [comment]
How does your question relate to this project?

**Reply by ghost** [reply]
I apologize for asking, but I haven't been able to find any information online about this particular topic. It seems to be a relatively new concept. Could you please help me understand if it's possible to use this webadb to grant permissions like software does? Additionally, I'm curious if they have similar command functionalities. Thank you for your understanding.

**Comment by yume-chan** [comment]
`adb shell pm grant com.arlosoft.macrodroid android.permission.READ_LOGS` works only if the app has declared the permission in its manifest. Usually the app developer will tell you to do that. Running this command for any arbitrary app has no effect.

If it works, it's same as running `m grant com.arlosoft.macrodroid android.permission.READ_LOGS` in the Interactive Shell of the demo website.
